scripts/retdec.py

```diff
@@ -48,8 +48,8 @@ def do_invoke(self, argv):
             "raw_endian": "big" if is_big_endian() else "little",
         }
 
-        raw_cmd = "{}  -m {} --raw-section-vma {} --raw-entry-point {} -e {} -f plain -a {} -o {} -l {} {} --cleanup"
-        bin_cmd = "{}  -m {} -e {} -f plain -a {} -o {} -l {} {} --cleanup"
+        raw_cmd = "{}  -m {} --raw-section-vma {} --raw-entry-point {} -e {} -f plain -a {} -o {} -l {} '{}' --cleanup"
+        bin_cmd = "{}  -m {} -e {} -f plain -a {} -o {} -l {} '{}' --cleanup"
```

Comment:
The first format parameter is also a path (the path of retdec). Would be good to wrap it with quotes too.

Reply:
Made the change! Thanks for letting me know :D